.gitignore

```diff
@@ -0,0 +1,163 @@
+data/*.csv
```

Comment:
So just delete those changes, and add it to the end of the `.gitignore` file, under `### Custom ###` (if we don't have that section in the file, add it at the end).